cyber-security/modules/csls_s3_object_logging/s3.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_s3_bucket" "s3-object-logging" {
+  bucket        = local.cloudtrail_bucket_name
+  force_destroy = true
+
+  tags = merge(var.tags, map("Name", local.cloudtrail_bucket_name))
+
+  policy = <<POLICY
```

Comment:
again, not sure if the heredo can be replaced with a data template

Reply:
Done